packages/@okta/vuepress-site/docs/guides/mobile-idx-sdk-overview/main/index.md

```diff
@@ -0,0 +1,165 @@
+
+![A diagram showing the SDK objects and their relationships.](/img/mobile-sdk/mobile-idx-objects.png "A diagram that shows the SDK objects for the sign in flow and the relationships between them.")
+
+</div>
+
+## Common parts of the flow
+
+Some parts of the sign-in flow are the same:
+
+- Add the SDK to your app.
+- Configure the SDK.
+- Initialize the client and start the flow.
+- Process the response.
+- Request a token.
+- Sign-out the user.
+
+### Add the SDK
+
+<StackSnippet snippet="adddependency" />
+
+### Create and manage configurations
+
+A _Configuration_ contains the settings used by the SDK to connect to an Okta Application Integration:
+
+| Value         | Description |
+| :------------ | :---------- |
+| Issuer        | The Oauth2 URL for the Okta Org, such as `https://oie-123456.oktapreview.com/oauth2/default`. |
+| Client ID     | The ID of the Okta Application Integration from the Okta Admin console.  |
+| Client secret | An optional shared secret for accessing the Application Integration. |
+| Redirect URI  | A callback URI for launching the mobile app, such as `com.example.oie-123456:/callback`. |
+| Scopes        | A space separated list of the requested access scopes for the app. |
+
+The configuration information in a shipping app is usually static. You can initialize the
+configuration values directly in the code or by reading them in from a file. During
+development you may want to provide a way to edit configuration values.
+
+<StackSnippet snippet="loadingaconfiguration" />
+
+### Start sign-in
+
+Start the sign-in flow by first creating an SDK client using a configuration, and then requesting the first step.
+
+<StackSnippet snippet="initializingsdksession" />
+
+### Process the response
+
+The steps for processing a response are:
+1. Check for a successful sign-in.
+1. Check for messages, such as an invalid password.
+1. Check for remediations.
+1. Process the remediations.
+1. Process the authenticators.
+
+After the user enters the required information, update the remediation and request the next step.
+
+<StackSnippet snippet="processresponse" />
+
+### Complete sign in
+
+Check the response for a successful sign-in at each step of the flow. When the user is
+signed-in, exchange the session token for an access token, and then exit the flow.
+
+<StackSnippet snippet="gettingatoken" />
+
+### Sign out a user
+
+Sign out a user by revoking the access token granted by signing in, and by revoking a
+refresh token if one exists.
+
+<StackSnippet snippet="signingout" />
+
+## Sign-in flow UI
+
+There are two general approaches for adding the sign-in flow to your app: add a fixed
+number of authentication methods possibly in a fixed order, or generate the user interface
+dynamically based on the response at each step.
+
+For a fixed number of methods, you create appropriate views and populate them from the
+response. Using fixed methods does introduce risk as authentication methods may be added
```

Comment:
I used that order of concepts intentionally. I want to the developer journey for that sentence to be:
"You can use a static UI but doing so has risk because not just the order of methods, but the methods themselves can change at any time by someone else."

Thoughts?

Reply:
I like what you said above better than what you have. :)

Reply:
Changed it to read:

> For a fixed number of methods (a static UI), you create appropriate views and populate them from the response. Using a static UI introduces risk as both the authentication methods and the order may be changed at any time by an administrator of your Okta org.